src/Construct.php

```diff
@@ -206,11 +208,18 @@ protected function docs()
     /**
      * Generate README.md file.
      *
+     * @param boolean $withGitHubTemplates Whether or not to set a reference to the .github/contributing file in the readme.
+     *
      * @return void
      */
-    protected function readme()
+    protected function readme($withGitHubTemplates = false)
```

Comment:
Maybe you can use `$this->settings->withGithubTemplates()` in this `readme()` method directly? instead of pass it around.